app/views/profile/settings.html.erb

```diff
@@ -0,0 +1,77 @@
+<div class="uk-width-medium-1-2 uk-container-center">
+    <%= form_tag({action: :save_settings}, {method: :post}) do %>
+        <table class="uk-table uk-table-striped uk-table-hover">
+            <% for attribute in attributes do %>
+                <tr>
+                    <td><label><%= attribute[:text] %><%= " *" if attribute[:required] %></label></td>
+                    <td>
+                        <%=
+                            if attribute[:type] == :text
+                                text_field_tag attribute[:name], @user["usr_" + attribute[:name]], required: attribute[:required]
+                            elsif attribute[:type] == :select
+                                select_tag attribute[:name], options_for_select(attribute[:options], selected: @user["usr_" + attribute[:name]])
+                            end
+                        %>
+                    </td>
+                    <td>
+                    <% if attribute[:visibility_settings] %>
+                        <%= visibility_select(attribute[:name]) %>
+                    <% end %>
+                    </td>
+                </tr>
+            <% end %>
+        </table>
+
+        <!-- IM accounts -->
+        <table>
+            <tr>
+                <td><label>IM</label></td>
+                <td>
+                    <table>
+                        <tr>
+                            <td colspan="2"><input type="submit" class="uk-button" value="Új"/></td>
+                        </tr>
+                    </table>
+                </td>
+            </tr>
+            <tr>
+                <td><label>Neptun</label></td>
+                <td><span><%= @user.neptun %> </span></td>
+            </tr>
+        </table>
+        <div class="uk-text-center">
+            <div class="inline-block">
+                <input class="uk-button uk-button-success" type="submit" value="Módosítás" name="submitButton"/>
+            </div>
+        </div>
+    <% end %>
+
+    <!-- Profile picture upload and cropping -->
+    <div class="uk-margin-large-top uk-margin" id="image-upload-container">
+        <%= form_tag({action: :settings}, multipart: true) do %>
```

Comment:
Itt is erdemes lenne a kepfeltoltesnek egy sajat form objectet letrehozni Es azt hasznalni itt.